build.gradle.kts

```diff
@@ -1,25 +1,28 @@
 import com.vanniktech.maven.publish.MavenPublishBaseExtension
 import com.vanniktech.maven.publish.SonatypeHost
 import java.net.URL
+import nl.littlerobots.vcu.plugin.versionCatalogUpdate
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.jetbrains.dokka.gradle.DokkaTask
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import ru.vyarus.gradle.plugin.animalsniffer.AnimalSnifferExtension
 
 buildscript {
   dependencies {
-    classpath(Dependencies.kotlinPlugin)
-    classpath(Dependencies.dokkaPlugin)
-    classpath(Dependencies.androidPlugin)
-    classpath(Dependencies.androidJunit5Plugin)
-    classpath(Dependencies.graalPlugin)
-    classpath(Dependencies.bndPlugin)
-    classpath(Dependencies.shadowPlugin)
-    classpath(Dependencies.japicmpPlugin)
-    classpath(Dependencies.animalsnifferPlugin)
-    classpath(Dependencies.errorpronePlugin)
-    classpath(Dependencies.spotlessPlugin)
-    classpath(Dependencies.vanniktechPublishPlugin)
+    classpath(libs.gradleplugin.dokka)
+    classpath(libs.gradleplugin.kotlin)
+    classpath(libs.gradleplugin.android.junit5)
+    classpath(libs.gradleplugin.android)
+    classpath(libs.gradleplugin.palantir.graal)
+    classpath(libs.gradleplugin.aqute.bnd)
+    classpath(libs.gradleplugin.johnrengelman.shadow)
+    classpath(libs.gradleplugin.japicmp)
+    classpath(libs.gradleplugin.animalsniffer)
+    classpath(libs.gradleplugin.errorprone)
+    classpath(libs.gradleplugin.spotless)
+    classpath(libs.gradleplugin.vanniktech)
+    classpath(libs.gradleplugin.benmanes.versions)
+    classpath(libs.gradleplugin.littlerobots.vcu)
```

Comment:
Suggest removing these 2 plugins after migration.

Reply:
Why? They simplify the update process. For me this is more valuable than just refactoring our dependency files.

Reply:
Once you finish the migration, the `littlerobots.vcu` is less helpful, if you want to check updates, [using a Gradle init script](https://github.com/ben-manes/gradle-versions-plugin#using-a-gradle-init-script) might be a better solution; if you want to update automatically, can use [renovate](https://www.whitesourcesoftware.com/free-developer-tools/renovate/).

Reply:
Is there a good sample of the former? It appears to have the same effect, but requires editing files that we currently currently .gitignore.

Reply:
We're doing renovate on cashapp org. It's... fine. Will hold us over until dependabot decides to wake up and actually support other ecosystems.

Reply:
> Is there a good sample of the former? It appears to have the same effect, but requires editing files that we currently currently .gitignore.

detekt/detekt/pull/4294

Reply:
OK, I'll remove the plugins, and wait for someone in Block to configure for this repo.

I can see them coming through now https://github.com/yschimke/okhttp/pulls

Reply:
Involuntary shudder reading "someone at Block"... 🤮

Reply:
@yschimke Missed removing this?

Reply:
Yep - fixed here https://github.com/square/okhttp/pull/7081/files